dagger-py-functions/udfs/scalar/sample_udf.py

```diff
@@ -0,0 +1,17 @@
+from pyflink.table import ScalarFunction, DataTypes
+from pyflink.table.udf import udf
+
+class SampleUdf(ScalarFunction):
+
+    def __init__(self):
+        self.counter = None
+
+    def open(self, function_context):
+        mg = function_context.get_metric_group()
+        self.counter = mg.add_group("udf", "SampleUdf").counter("value")
```

Comment:
what exactly do we want to publish to metrics?

Reply:
removed the metric part from the code.